### PR TITLE
Add unit test for heightmanager

### PR DESCRIPTION
**Pull request by lzxm160**
1、add test  heightManager add
2、add test  heightManager nearestHeightBefore
3、add test  heightManager lastestHeight
4、add test  heightManager validate

### CODE REVIEW

File: committee/heightmanager_test.go [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+package committee
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+type arg struct {
```
Comment (CoderZhi) [inline committee/heightmanager_test.go]: add space between different blocks

File: committee/heightmanager_test.go [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+		time.Unix(int64(1546272001), 0),
+		time.Unix(int64(1546272016), 0),
+		time.Unix(int64(1546272031), 0),
+		time.Unix(int64(1546272046), 0),
+		time.Unix(int64(1546272061), 0),
+	}
+)
+func TestNewHeightManager(t *testing.T) {
+	require := require.New(t)
+	hm := newHeightManager()
```
Comment (CoderZhi) [inline committee/heightmanager_test.go]: ```require.NotNil(newHeightManager())```

File: committee/heightmanager_test.go [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+	require.NotNil(hm)
+}
+
+func TestAdd(t *testing.T) {
+	require := require.New(t)
+	hm := newHeightManager()
+	var err error
+	for _,arg:=range args{
+		err=hm.add(arg.height,arg.time)
```
Comment (CoderZhi) [inline committee/heightmanager_test.go]: ```require.NoError(hm.add(arg.height, arg.time))```

same for the rest

**Comment by lzxm160** [comment]
merge with 37

**Pull request closed without merging**